i18n-wg/proposed-charter-2019-2020.html

```diff
@@ -162,7 +162,16 @@ <h2>
   <h3>
             Normative Specifications
           </h3>
-          <p>The formal documents produced by the Working Group are guidelines, best practices, requirements, and the like. These are best published as Working Group Notes. The Working Group will not produce Rec-track specifications.</p>
+          <p>The formal documents produced by the Working Group are guidelines, best practices, requirements, and the like. These are best published as Working Group Notes. The Working Group will not produce Rec-track specifications, with the exception
+            of maintaining the following W3C Recommendations:</p>
+            <dl>
+              <dt>Internationalization Tag Set (ITS) <a href="https://www.w3.org/TR/2007/REC-its-20070403/">Version 1.0</a> and <a href="http://www.w3.org/TR/its20/">Version 2.0</a></dt>
+              <dd>These spoecifications define data categories and their implementation as a set of elements and attributes. ITS 2 adds additional concepts that are designed to foster the automated creation and processing of multilingual Web content.</dd>
+              <dt><a href="http://www.w3.org/TR/charmod/">Character Model for the World Wide Web 1.0: Fundamentals</a> </dt>
+              <dd>It provides authors of specifications, software developers, and content developers with a common reference for interoperable text manipulation on the World Wide Web</dd>
+              <dt><a href="http://www.w3.org/TR/ruby">Ruby Annotation</a></dt>
+              <dd>"Ruby" are short runs of text alongside the base text, typically used in East Asian documents to indicate pronunciation or to provide a short annotation. This specification defines markup for rub.</dd>
```

Comment:
rub -> ruby

Reply:
With Richard's nit, the second revision looks good to me. I double-checked the other publications I remember and all are Note status.